packages/common/src/projects/_internal/ProjectSchema.ts

```diff
@@ -51,6 +51,18 @@ export const ProjectSchema: IConfigurationSchema = {
         timeline: {
           type: "object",
         },
+        tags: {
```

Comment:
`tags` and `categories` live at the top-level of the entity, so we shouldn't nest them under `view` in the `schema`